jest.config.js

```diff
@@ -1,5 +1,8 @@
 /** @type {import('ts-jest').JestConfigWithTsJest} */
-module.exports = {
+export default {
   preset: "ts-jest",
   testEnvironment: "node",
+  moduleNameMapper: {
+    "^(\\.\\.?\\/.+)\\.jsx?$": "$1",
```

Comment:
jsx?

Reply:
the x is optional (`x?`), it's just the recommended regex but we can remove the `x`, I just used what most projects are using.